app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -187,6 +192,49 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_clickLogoutButton_displaysLogoutDialog() {
+    ActivityScenario.launch<AdministratorControlsActivity>(createAdministratorControlsActivityIntent(0)).use {
+      onView(withId(R.id.administrator_controls_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(4))
+      onView(withId(R.id.log_out_text_view)).perform(click())
+      onView(withText(R.string.log_out_dialog_message)).inRoot(isDialog()).check(matches(isDisplayed()))
+      onView(withText(R.string.log_out_dialog_okay_button)).inRoot(isDialog()).check(matches(isDisplayed()))
+      onView(withText(R.string.log_out_dialog_cancel_button)).inRoot(isDialog()).check(matches(isDisplayed()))
+    }
+  }
+
+  // TODO(#762): Replace ProfileActivity to LoginActivity once it is added.
+  @Test
+  fun testAdministratorControlsFragment_clickOkButtonInLogoutDialog_checkOpensProfileActivity() {
+    ActivityScenario.launch<AdministratorControlsActivity>(createAdministratorControlsActivityIntent(0)).use {
+      onView(withId(R.id.administrator_controls_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(4))
+      onView(withId(R.id.log_out_text_view)).perform(click())
+      onView(withText(R.string.log_out_dialog_message)).inRoot(isDialog()).check(matches(isDisplayed()))
+      onView(withText(R.string.log_out_dialog_okay_button)).perform(click())
+      intended(hasComponent(ProfileActivity::class.java.name))
+    }
+  }
+
+  @Test
+  fun testAdministratorControlsFragment_clickCancelButtonInLogoutDialog_checkDialodDismissed() {
+    ActivityScenario.launch<AdministratorControlsActivity>(createAdministratorControlsActivityIntent(0)).use {
+      onView(withId(R.id.administrator_controls_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(4))
+      onView(withId(R.id.log_out_text_view)).perform(click())
+      onView(withText(R.string.log_out_dialog_message)).inRoot(isDialog()).check(matches(isDisplayed()))
+      onView(withText(R.string.log_out_dialog_cancel_button)).perform(click())
+      onView(withId(R.id.log_out_text_view)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testAdministratorControlsFragment_clickAppVersion_checkOpensAppVersionActivity() {
```

Comment:
`checkOpensAppVersionActivity` should be replaced with `opensAppVersionActivity`.
The last information is the `expected_outcome` and therefore check should not be added here considering that `check` is still an step.

Reply:
Done